packages/vue-client/src/views/GameView.vue

```diff
@@ -26,22 +26,55 @@ const DEFAULT_GAME: GameResponse = {
   moves: [],
   config: {},
 };
+
+// null <-> viewing the latest round
+// while viewing history of game, maybe we should prevent player from making a move (accidentally)
+const view_round: Ref<number | null> = ref(null);
+const navigateRounds = (offset: number): void => {
+  if (game.value.round === undefined) {
+    // game not initialised
+    return;
+  }
+
+  const after_offset = (view_round.value ?? game.value.round) + offset;
+  const round_in_range = Math.min(game.value.round, Math.max(0, after_offset));
+
+  view_round.value =
+    round_in_range === game.value.round ? null : round_in_range;
+};
+
 const gameResponse: GameResponse = reactive(DEFAULT_GAME);
 const game = computed(() => {
   if (!gameResponse.variant) {
     return { result: null, state: null };
   }
   const game_obj = makeGameObject(gameResponse.variant, gameResponse.config);
+
+  let state: unknown = null;
   gameResponse.moves.forEach((m) => {
```

Comment:
Maybe that would be better, I don't know. Here are some reasons why it might be preferable to continue the loop:
1.) we have the game result (if finished) also while navigate through moves.
2.) we know the latest round number, so we could display something like "round: 23 / 31". I haven't added this because I don't know where and how to position all the elements on the game view page.